android/java/org/chromium/chrome/browser/rate/BraveAskPlayStoreRatingDialog.java

```diff
@@ -0,0 +1,123 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.rate;
+
+import android.annotation.SuppressLint;
+import android.app.Dialog;
+import android.content.Context;
+import android.os.Bundle;
+import android.os.Handler;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.Button;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.AppCompatButton;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
+
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
+import com.google.android.play.core.review.ReviewInfo;
+import com.google.android.play.core.review.ReviewManager;
+import com.google.android.play.core.review.ReviewManagerFactory;
+import com.google.android.play.core.tasks.Task;
+
+import org.chromium.chrome.R;
+
+public class BraveAskPlayStoreRatingDialog extends BottomSheetDialogFragment {
+    final public static String TAG_FRAGMENT = "brave_ask_play_store_rating_dialog_tag";
+    private static final String TAG = "BraveAskPlayStoreRating";
+
+    public static BraveAskPlayStoreRatingDialog newInstance() {
+        return new BraveAskPlayStoreRatingDialog();
+    }
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setStyle(STYLE_NORMAL, R.style.AppBottomSheetDialogTheme);
```

Comment:
@sujitacharya2005 have you verified the change on android 6 ? i want to make sure for style on android 6

Reply:
same style is already used in BraveStatsBottomSheetDialogFragment.java. 
I will re-verify it.